### PR TITLE
AdminUI : add afsearchPrintPagePDFFormats

### PR DESCRIPTION
**Pull request by UshaMakoa**
Overview
----------------------------------------
AdminUI : add afsearchPrintPagePDFFormats

### CODE REVIEW

**Comment by breheret** [comment]
test this please

File: ext/civicrm_admin_ui/ang/afsearchPrintPagePDFFormats.aff.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+use CRM_CivicrmAdminUi_ExtensionUtil as E;
+
+return [
+  'type' => 'search',
+  'title' => E::ts('Print Page (PDF) Formats'),
+  'icon' => 'fa-list-alt',
+  'server_route' => 'civicrm/admin_ui/admin/pdfFormats',
+  'modified_date' => '2023-12-05 08:10:02',
```
Comment (colemanw) [inline ext/civicrm_admin_ui/ang/afsearchPrintPagePDFFormats.aff.php:9]: Pls delete this line

File: ext/civicrm_admin_ui/managed/SavedSearch_Print_Page_PDF_Formats.mgd.php [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+              'path' => 'civicrm/admin/pdfFormats?action=add&reset=1',
+              'icon' => 'fa-external-link',
+              'text' => E::ts('Add PDF Page Format'),
+              'style' => 'default',
+              'condition' => [],
+              'task' => '',
+              'entity' => '',
+              'action' => '',
+              'join' => '',
+              'target' => '',
```
Comment (colemanw) [inline ext/civicrm_admin_ui/managed/SavedSearch_Print_Page_PDF_Formats.mgd.php:169]: Typically we create a new item in a popup:

```suggestion
              'target' => 'crm-popup',
```

File: ext/civicrm_admin_ui/managed/SavedSearch_Print_Page_PDF_Formats.mgd.php [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+              'size' => 'btn-xs',
+              'icon' => 'fa-bars',
+              'links' => [
+                [
+                  'entity' => 'OptionValue',
+                  'action' => '',
+                  'join' => '',
+                  'target' => 'crm-popup',
+                  'icon' => 'fa-toggle-on',
+                  'text' => E::ts('Enable Option Values'),
```
Comment (colemanw) [inline ext/civicrm_admin_ui/managed/SavedSearch_Print_Page_PDF_Formats.mgd.php:129]:
```suggestion
                  'text' => E::ts('Enable'),
```

File: ext/civicrm_admin_ui/managed/SavedSearch_Print_Page_PDF_Formats.mgd.php [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+                  'task' => 'enable',
+                  'condition' => [],
+                ],
+                [
+                  'task' => 'disable',
+                  'entity' => 'OptionValue',
+                  'join' => '',
+                  'target' => 'crm-popup',
+                  'icon' => 'fa-toggle-off',
+                  'text' => E::ts('Disable Option Values'),
```
Comment (colemanw) [inline ext/civicrm_admin_ui/managed/SavedSearch_Print_Page_PDF_Formats.mgd.php:141]:
```suggestion
                  'text' => E::ts('Disable'),
```

File: ext/civicrm_admin_ui/managed/SavedSearch_Print_Page_PDF_Formats.mgd.php [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+                  'style' => 'warning',
+                  'path' => '',
+                  'action' => '',
+                  'condition' => [],
```
Comment (colemanw) [inline ext/civicrm_admin_ui/managed/SavedSearch_Print_Page_PDF_Formats.mgd.php:145]:
```suggestion
                  'condition' => ['is_active', '=', TRUE],
```

File: ext/civicrm_admin_ui/managed/SavedSearch_Print_Page_PDF_Formats.mgd.php [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+                  'entity' => 'OptionValue',
+                  'action' => '',
+                  'join' => '',
+                  'target' => 'crm-popup',
+                  'icon' => 'fa-toggle-on',
+                  'text' => E::ts('Enable Option Values'),
+                  'style' => 'default',
+                  'path' => '',
+                  'task' => 'enable',
+                  'condition' => [],
```
Comment (colemanw) [inline ext/civicrm_admin_ui/managed/SavedSearch_Print_Page_PDF_Formats.mgd.php:133]:
```suggestion
                  'condition' => ['is_active', '=', FALSE],
```

File: ext/civicrm_admin_ui/managed/SavedSearch_Print_Page_PDF_Formats.mgd.php [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+              'links' => [
+                [
+                  'entity' => '',
+                  'action' => '',
+                  'join' => '',
+                  'target' => 'crm-popup',
+                  'icon' => 'fa-external-link',
+                  'text' => E::ts('Edit'),
+                  'style' => 'default',
+                  'path' => 'civicrm/admin/pdfFormats?action=update&id=[id]&reset=1',
```
Comment (colemanw) [inline ext/civicrm_admin_ui/managed/SavedSearch_Print_Page_PDF_Formats.mgd.php:97]:
```suggestion
                  'entity' => 'OptionValue',
                  'action' => 'update',
                  'join' => '',
                  'target' => 'crm-popup',
                  'icon' => 'fa-external-link',
                  'text' => E::ts('Edit'),
                  'style' => 'default',
```

File: ext/civicrm_admin_ui/managed/SavedSearch_Print_Page_PDF_Formats.mgd.php [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+                  'icon' => 'fa-trash',
+                  'text' => E::ts('Delete'),
+                  'style' => 'danger',
+                  'path' => '',
+                  'task' => '',
+                  'condition' => [],
+                ],
+              ],
+              'type' => 'buttons',
+              'alignment' => 'text-right',
```
Comment (colemanw) [inline ext/civicrm_admin_ui/managed/SavedSearch_Print_Page_PDF_Formats.mgd.php:115]: This split with some links in a "buttons" group and others in a "menu" group isn't typical. The convention in Admin UI is to place all links in a single "menu" group.

**Comment by samuelsov** [comment]
@UshaMakoa Just to let you know, I will try to finish this PR today unless you want to finish it.

**Comment by samuelsov** [comment]
Replacing this by a https://github.com/civicrm/civicrm-core/pull/29545

**Pull request closed without merging**